docs/spec/openapi.yml

```diff
@@ -0,0 +1,3602 @@
+openapi: "3.0.2"
+info:
+  title: GuideWire insuranceNow cloud API
+  version: 5.0.0
```

Comment:
Here we better add a high-level description, which is a common practice in well-written openAPI contracts

Reply:
Added a description with [27d693300fa4e0c1efec5f6911c0d1603a5a0317](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/2/commits/27d693300fa4e0c1efec5f6911c0d1603a5a0317)